auth0_flutter/darwin/Classes/AuthAPI/AuthAPIHandler.swift

```diff
@@ -55,6 +59,10 @@ public class AuthAPIHandler: NSObject, FlutterPlugin {
         case .userInfo: return AuthAPIUserInfoMethodHandler(client: client)
         case .renew: return AuthAPIRenewMethodHandler(client: client)
         case .resetPassword: return AuthAPIResetPasswordMethodHandler(client: client)
+        case .passwordlessWithEmail: return AuthAPIEmailPasswordlessLoginMethodHandler(client: client)
+        case .passwordlessWithPhoneNumber: return AuthAPIPhoneNumberPasswordlessLoginMethod(client: client)
```

Comment:
These should probably be named as follows:
- `AuthAPIPasswordlessEmailMethodHandler`
- `AuthAPIPasswordlessPhoneNumberMethodHandler`